src/utilstrencodings.cpp

```diff
@@ -33,6 +33,17 @@ string SanitizeString(const string& str, int rule)
     return strResult;
 }
 
+/// Formats the network peer user agent text (or subversion)
```

Comment:
Please put the comment in the h file. And use the formatting according to the guideline (CONTRIBUTING.md)